### ISSUE TITLE
Handling large files

### ISSUE THREAD

**Issue by raquel-ucl** [oracc/nammu#225]
Related to #157

Greta reported:

```
I've just tried to open the admittedly enormous file of sources to CMAwR 1, 2 and Maqlu with 
Nammu. But Nammu freezes instead... OK, the file is 1 MB, but Emacs can handle it without 
problems.
```

This is the file (txt extension added so GH lets me upload it here):

[cmawro12maqlu.atf.txt](https://github.com/oracc/nammu/files/905375/cmawro12maqlu.atf.txt)

I've tried opening the file and in fact it takes a long while to load. With the syntax highlight off, it takes 2 seconds on my machine, but that could still be improved. With syntax highlight on, I can't tell because I wasn't patient enough to wait and had to kill Nammu. 

Apart from fixing the syntax highlight performance, I could also try other things. None of JEditPane, JTextArea or JTextPane are very performant at opening files larger than 0.5MB. I'm thinking of doing some kind of lazy loading so that I just load at a given time the text that is being displayed and refresh upon scroll up/down. This won't work easily with the current syntax highlight though.

I'll think about all this when I'm back from holidays.

**Comment by sgrieve** [comment]
See issue #262 for how this has been resolved.

File loading is still slow, but once a big file is loaded, editing is responsive and the editor does not lock up.

**Comment by sgrieve** [comment]
File loading is no longer slow, see the latest changes in #262.

This issue can be closed once pr #262 is merged.
